### PR TITLE
Add Python3 compat

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: bin/vomses-crosscheck [hunk 1]
```diff
@@ -9,10 +9,10 @@ vomses_path = "vomses"
 vomsdir     = "vomsdir"
 
 def usage():
-    print "usage: %s [vomses vomsdir]" % os.path.basename(__file__)
-    print "   or: %s --system" % os.path.basename(__file__)
-    print
-    print "Use --system for /etc/vomses and /etc/grid-security/vomsdir"
+    print("usage: %s [vomses vomsdir]" % os.path.basename(__file__))
+    print("   or: %s --system" % os.path.basename(__file__))
+    print()
```
Comment (edquist) [inline bin/vomses-crosscheck]:
```suggestion
    print("")
```

File: bin/vomses-crosscheck [hunk 2]
```diff
@@ -83,6 +83,6 @@ for lsc_path in sorted(glob.glob("*/*.lsc")):
 for x in (missing_lsc_files, missing_vomses_entries, dn_mismatches):
     if x:
         for line in x:
-            print line
-        print
+            print(line)
+        print()
```
Comment (edquist) [inline bin/vomses-crosscheck]:
```suggestion
        print("")
```

**Comment by edquist** [comment]
Just remember `print()` in python2 (which this developer still cares about) prints `()`.

**Comment by matyasselmeci** [comment]
I ran 2to3 on the program; I'm surprised it didn't add the `from __future__ import print_function` line, but I think that's the proper solution.

**Comment by edquist** [comment]
Yeah that's fine too.  I thought maybe you were avoiding it since `print(...)` is legal python2 also.

**Pull request merged**